lib/migrate.js

```diff
@@ -46,11 +46,10 @@ function migrate (set, direction, migrationName, fn) {
     if (isPromise) {
       returnValue
         .then(completeMigration)
-        .catch(completeMigration)
+        .catch(fn)
     }
 
-    function completeMigration (err) {
-      if (err) return fn(err)
+    function completeMigration () {
```

Comment:
There is a call to `completeMigration(err)` a few lines up, so that probably needs to still be here

Reply:
Yeah, keep the error handling as it was.  Only the change to call `fn` on catch is needed.